pkg/env/vulnerability_management.go

```diff
@@ -3,4 +3,10 @@ package env
 var (
 	// ReportExecutionMaxConcurrency sets the maximum number vulnerability reports that can run in parallel
 	ReportExecutionMaxConcurrency = RegisterIntegerSetting("ROX_REPORT_EXECUTION_MAX_CONCURRENCY", 5)
+
+	// ReportCustomEmailSubjectMaxLen sets the maximum allowed length for custom email subject in vulnerability report emails
+	ReportCustomEmailSubjectMaxLen = RegisterIntegerSetting("ROX_REPORT_CUSTOM_EMAIL_SUBJECT_MAX_LEN", 256)
+
+	// ReportCustomEmailBodyMaxLen sets the maximum allowed length for custom email body in vulnerability report emails
+	ReportCustomEmailBodyMaxLen = RegisterIntegerSetting("ROX_REPORT_CUSTOM_EMAIL_BODY_MAX_LEN", 1500)
```

Comment:
@dvail I added env vars to control the maximum allowed length for custom email subject and body when configuring vuln reports. Could you also update the UI to use the same env vars instead of constants?

For context, previously we used agreed upon values but that way we need to make sure it remains consistent between BE and FE. Also, it did not provide flexibility to the user to increase the limits.